<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>B16SortTestBy04200923</string>
  </property>
  <widget class="QPushButton" name="startPushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QTableView" name="ResultTableView">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>531</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="getrandarry_newCheckBox">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>241</y>
     <width>237</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>getrandarry_new</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="getrandarry_oldCheckBox">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>278</y>
     <width>237</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>getrandarry_old</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="getSequentialArryCheckBox">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>315</y>
     <width>265</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>getSequentialArry</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>33</y>
     <width>228</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="TimesLable">
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;time:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="timeSpinBox">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="AmountLabel">
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;amount:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="AmountSpinBox">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="LvalueLabel">
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;lvalue:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="lvalueSpinBox">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="RvalueLabel">
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;rvalue:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="rvalueSpinBox">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="ResultPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>851</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
